controller/lifecycle/lifecycle.go

```diff
@@ -125,23 +163,25 @@ func (l *LifecycleManager) Reconcile(ctx context.Context, req ctrl.Request, inst
 				updateObservedGeneration(instanceStatusObj, log)
 			}
 		}
-	}
 
-	currentStatus := reflect.Indirect(reflect.ValueOf(instance)).FieldByName("Status").Interface()
-	originalStatus := reflect.Indirect(reflect.ValueOf(c)).FieldByName("Status").Interface()
-	equal := reflect.DeepEqual(currentStatus, originalStatus)
-	if !equal {
-		log.Info().Msg("updating resource status")
-		err = l.client.Status().Update(ctx, instance)
-		if err != nil {
-			if !k8sErrors.IsConflict(err) {
-				sentry.CaptureError(err, sentryTags, sentry.Extras{"message": "Updating of instance status failed"})
-			}
-			log.Error().Err(err).Msg("cannot update reconciliation Conditions, kubernetes client error")
-			return result, err
+		if l.manageConditions {
+			setInstanceConditionReady(&conditions, v1.ConditionTrue)
 		}
 	} else {
-		log.Info().Msg("skipping status update, since they are equal")
```

Comment:
🟡  i would be for keeping this log statement in the else branch for now, since we might not be fast enough to migrate everywere

Reply:
then i'm not sure i get your comment, just keep it as is?

Reply:
yes i missed it